src/commands/createWebApp/setDefaultRgAndPlanName.ts

```diff
@@ -48,16 +52,16 @@ export async function setDefaultRgAndPlanName(wizardContext: IAppServiceWizardCo
             }
 
             // otherwise create a new rg and asp
-            wizardContext.newResourceGroupName = wizardContext.newResourceGroupName || await siteNameStep.getRelatedName(wizardContext, defaultName);
+            wizardContext.newResourceGroupName = wizardContext.newResourceGroupName || await siteNameStep.getRelatedName(wizardContext, defaultGroupName);
```

Comment:
A new rg wouldn't be created until the app service quota is met which looks to be 3 app services in one rg

Reply:
Good point. That being said, I believe it's 3 app services in one _plan_ - I don't think the rg matters as much. Which means it would be more important to use `defaultPlanName` when calling `getRelatedName`